reporter/src/funTest/kotlin/reporters/spdx/SpdxDocumentReporterFunTest.kt

```diff
@@ -168,7 +168,7 @@ private val ortResult = OrtResult(
     ),
     analyzer = AnalyzerRun(
         environment = Environment(),
-        config = AnalyzerConfiguration(allowDynamicVersions = true),
+        config = AnalyzerConfiguration(),
```

Comment:
Mind squashing this to the previous commit? I believe it's easier to review the same change in a single blow, and it avoids you to retype the same commit message.